lib/galaxy/config/sample/subdomains_conf.yml.sample

```
@@ -0,0 +1,8 @@
+# Each subdomain entry will be rendered as a link in the navbar
+# Ensure that you add an entry for your base site
+
+subdomains:
+  - title: Base
+    url: https://mygalaxy.org
```

This is an existing and parked domain. 

```suggestion
    url: https://mygalaxy.example.com/
```